src/cli/commands/snip.ts

```diff
@@ -72,6 +130,29 @@ const handler = async ({
     }
   });
 
+  if (format !== undefined) {
```

Comment:
This should check specifically which formatter the user wanted.
```suggestion
  if (format === "sphynx-rst") {
```

Reply:
You could also report if the format string was not accepted.